tensorflow/lite/python/lite.py

```diff
@@ -348,7 +348,8 @@ def from_keras_model_file(cls,
                             model_file,
                             input_arrays=None,
                             input_shapes=None,
-                            output_arrays=None):
+                            output_arrays=None,
+                            custom_objects=None):
```

Comment:
Instead of adding `custom_objects` is it possible to add `**kwargs` to `from_keras_model_file()`?. See below for rest of the comments.